src/Main.java

```diff
@@ -20,59 +21,50 @@ public static void main(String[] args) {
             System.out.print(" "+t+" ");
         }
         int y =12_000_000;
-        int people =12_000_000/1000;
         int born=17;
         int dead=8;
-
-        for(int t=1;t<=10;t++)
+        for( int year=1;year<=10;year++)
         {
-            y=y+(people*(born -dead));
-            System.out.println("год "+t+" численность населения состовляет "+y);
+            y=y+(y*(born -dead));
```

Comment:
```suggestion
            y=y+(y*(born -dead)?1000);
```
нужно делить на 1000 ещё